content/enterprise_influxdb/v1/about-the-project/release-notes.md

```diff
@@ -21,13 +168,13 @@ menu:
 
 #### Flux updates
```

Comment:
Do we want to discuss the new support for this argument in the latest flux (1.11.2 and onwards): https://docs.influxdata.com/flux/v0/stdlib/universe/histogramquantile/#onnonmonotonic?